README.md

```diff
@@ -1,24 +1,157 @@
-# README
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] It will be great to ensure that the link to your logo is not broken. This will make your work even more professional.